app/views/articles/_related_videos.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class="youtube-videos">
+  <%= render "liquids/youtube", id: "rvfmPtkhqok", width: 766, height: 431 %>
+  <%= render "liquids/youtube", id: "-Mkp4oavz2M", width: 766, height: 431 %>
+  <%= render "liquids/youtube", id: "0Atk61qbLT0", width: 766, height: 431 %>
+</div>
+<script>
+  const options = {
+    method: 'POST',
```

Comment:
And do we need to change this to a `'GET'`?

Reply:
Yes Jomah, and our fetch call should be a lot simpler to write out. With it being a GET we can now just make the fetch call, pass the params, and json the response, no need to pass the 'options' with the method type and headers.